tests/acceptance/10_files/02_maintain/changes_update_hashes.cf

```diff
@@ -218,8 +218,8 @@ TEST.cfengine.noupdate.updated,C,Content changed");
                     "!test_noupdate_same_repaired",
                     "test_noupdate_same_kept",
                     "!test_noupdate_same_notkept",
-                    "!test_noupdate_updated_repaired",
-                    "test_noupdate_updated_notkept",
+                    "test_noupdate_updated_repaired", # because changes are recorded in the changes log
```

Comment:
It might be worth changing the names of the changes body to illustrate the fact that hashes are not updated, but content db is (`because report_changes => "all"`).

Maybe, `test_changes_noupdate` -> `test_changes_noupdatehashes_report_changes_all`.